src/Foundation/NSMetadataItem.cs

```diff
@@ -18,12 +18,34 @@ bool GetBool (NSString key)
 			return n == null ? false : n.BoolValue;
 		}
 
+		bool? GetNullableBool (NSString key)
+		{
+			var n = Runtime.GetNSObject<NSNumber> (GetHandle (key));
+			return n?.BoolValue;
+		}
+
 		double GetDouble (NSString key)
 		{
 			var n = Runtime.GetNSObject<NSNumber> (GetHandle (key));
 			return n == null ? 0 : n.DoubleValue;
 		}
 
+		double? GetNullableDouble (NSString key)
+		{
+			var n = Runtime.GetNSObject<NSNumber> (GetHandle (key));
+			return n?.DoubleValue;
+		}
+
+		nint? GetNInt (NSString key)
```

Comment:
The only reason the others got the Nullable in the name is because it's replacing the existing versions so they couldn't just be GetDouble and GetBool, I figured since the nint version will be the only one so it shouldn't need the extra text in the name since it's private like sebastien said.